doc/modules/cassandra/pages/developing/cql/cql_singlefile.adoc

```diff
@@ -579,6 +579,14 @@ link:#compressionOptions[below].
 
 |`caching` |_map_ |_see below_ |Caching options, see
 link:#cachingOptions[below].
+
+|`crc_check_chance` |simple|1.0|When compression is enabled, each compressed
```

Comment:
This looks a bit incorrect. We do generate checksums for all data files, but we only check them on reads with compressed files (in part because it's impossible to do for the memory-mapped access mode). Perhaps

"This option defines the probability with which checksums should be checked during reads to detect bit rot and prevent the propagation of corruption to other replicas. The default value is 1 to apply a checksum every time a data chunk is read. Set to 0 to disable checksum checking and to 0.5 for instance to check every other read.

Due to technical limitations we only currently apply this for compressed files. If compression is not enabled on the table, no checksums will be verified."?

Reply:
sure, works for me.